src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String DATE = "dd.MM.yyyy";
+    private static final String REPORT = "Report for period ";
+    private static final String DASH = " - ";
+    private static final String SPACE = " ";
+    private static final String NEW_LINE = "\n";
+    private static final int SECOND_ELEMENT = 1;
+    private static final int THIRD_ELEMENT = 2;
+    private static final int FOURTH_ELEMENT = 3;
+    private static final int ZERO = 0;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder report = new StringBuilder();
+        LocalDate fromDate = LocalDate.parse(dateFrom, DateTimeFormatter.ofPattern(DATE));
+        LocalDate toDate = LocalDate.parse(dateTo, DateTimeFormatter.ofPattern(DATE));
+        int totalSalary = ZERO;
```

Comment:
В цьому випадку використання константи погіршує читабельність коду, а не полегшує її.
```suggestion
        int totalSalary = 0;
```